tasks/main.yml

```diff
@@ -1,9 +1,12 @@
 ---
-- unarchive:
-
-    src: https://download.dokuwiki.org/src/dokuwiki/dokuwiki-stable.tgz
-    dest: /srv/
-    remote_src: True
-
+- name: check if installed
```

Comment:
"Check whether dokuwiki is already installed"

Reply:
changed